test/fixtures/cli-plugin-local/.gitattributes

```diff
@@ -0,0 +1 @@
+*.md text=auto eol=lf
```

Comment:
So Metalsmith itself is somehow converting the newline characters to LF? That's really strange. It shouldn't be changing anything on its own with no plugins. I guess that's a separate issue though...